servers.go

```diff
@@ -57,6 +57,40 @@ type ServerTemplateDetails struct {
 	Name        string   `json:"name"`
 }
 
+// ServerPlatformTemplates
+type ServerPlatformTemplates struct {
+	KVM    []ServerPlatformTemplateDetails `json:"KVM"`
+	OpenVZ []ServerPlatformTemplateDetails `json:"OpenVZ"`
```

Comment:
```suggestion
```

I suggest we retire the OpenVZ features all together in this library starting now. 

The platform is out of support and no new servers can be added, not even by existing customers. December 2023 is the last date where the platform is operational at all.